server/controllers/boardController.js

```diff
@@ -0,0 +1,33 @@
+import Board from '../models/boardModel';
+import { buildResponse } from '../utils/helpers';
+import {} from '../models/issuesModel';
+
+const getBoardDetails = async (req, res) => {
+  const id = parseInt(req.params.id, 10);
+  if (typeof id !== 'number') {
+    res.status(400).send(buildResponse(false, `${req.params.id} should be a number`));
```

Comment:
This messages will be shown to the user, So please add messages that the user understands.
```suggestion
    res.status(400).send(buildResponse(false, `Invalid request to get board details!`));
```

Reply:
@VikalpP I completely forgot that these messages are gonna be displayed on UI as it is. I will correct them all.

Reply:
@nimish-gupta Is using console.log fine in the code or should we add some npm-logger and handle the log levels as well?